app/views/web/dashboard/users/index.html.haml

```diff
@@ -0,0 +1,30 @@
+%h2 Users Management
+.table-responsive
+  %table.table.table-striped.table-sm
+    %thead
+      %tr
+        %th Email
+        %th GitHub username
+        %th Roles
+        %th Status
+      - @users.each do |user|
+        %tr
+          %td= link_to user.email, dashboard_user_path(user)
+          %td= user.name
+          %td
+            %ul
+              - user.roles.each do |role|
+                %li
+                  = role.name
+          %td
+            - if current_user.has_role?(:stuff)
+              - if user.active?
+                = link_to user.state, delete_dashboard_user_path(user), method: :put,
+                                                                        class: "btn btn-success",
+                                                                        data: { confirm: 'Are you sure to disable?' }
+              - else
```

Comment:
we should also consider the `screening_completed` state.
At this state, we should display button that leads to Applicants page `#show` method, where we can review applicant's quiz results.